scripts/publish.sh

```diff
@@ -20,6 +20,7 @@ fi
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 WORKING_DIR="$DIR/.."
+RELEASE_ARCHIVE_NAME="datadog-cluster-agent-boshrelease"
```

Comment:
is the value correct? `datadog-agent-cluster-release` or `datadog-cluster-agent-boshrelease`

Reply:
the later, for consistency with the `datadog-agent-boshrelease`